docs-src/router/index.js

```diff
@@ -0,0 +1,245 @@
+        component: UiIconButton,
+        title: 'UiIconButton',
+        sourceUrl: 'src/UiIconButton.vue'
+    },
+    {
+        path: '/ui-menu',
+        component: UiMenu,
+        title: 'UiMenu',
+        sourceUrl: 'src/UiMenu.vue'
+    },
+    {
+        path: '/ui-modal',
+        component: UiModal,
+        title: 'UiModal',
+        sourceUrl: 'src/UiModal.vue'
+    },
+    {
+        path: '/ui-popover',
+        component: UiPopover,
+        title: 'UiPopover',
+        sourceUrl: 'src/UiPopover.vue'
+    },
+    {
+        path: '/ui-preloader',
+        component: UiPreloader,
+        title: 'UiPreloader',
+        sourceUrl: 'src/UiPreloader.vue'
+    },
+    {
+        path: '/ui-progress-circular',
+        component: UiProgressCircular,
+        title: 'UiProgressCircular',
+        sourceUrl: 'src/UiProgressCircular.vue'
+    },
+    {
+        path: '/ui-progress-linear',
+        component: UiProgressLinear,
+        title: 'UiProgressLinear',
+        sourceUrl: 'src/UiProgressLinear.vue'
+    },
+    {
+        path: '/ui-radio',
+        component: UiRadio,
+        title: 'UiRadio',
+        sourceUrl: 'src/UiRadio.vue'
+    },
+    {
+        path: '/ui-radio-group',
+        component: UiRadioGroup,
+        title: 'UiRadioGroup',
+        sourceUrl: 'src/UiRadioGroup.vue'
+    },
+    {
+        path: '/ui-ripple-ink',
+        component: UiRippleInk,
+        title: 'UiRippleInk',
+        sourceUrl: 'src/UiRippleInk.vue'
+    },
+    {
+        path: '/ui-select',
+        component: UiSelect,
+        title: 'UiSelect',
+        sourceUrl: 'src/UiSelect.vue'
+    },
+    {
+        path: '/ui-slider',
+        component: UiSlider,
+        title: 'UiSlider',
+        sourceUrl: 'src/UiSlider.vue'
+    },
+    {
+        path: '/ui-snackbar',
+        component: UiSnackbar,
+        title: 'UiSnackbar',
+        sourceUrl: 'src/UiSnackbarContainer.vue'
+    },
+    {
+        path: '/ui-switch',
+        component: UiSwitch,
+        title: 'UiSwitch',
+        sourceUrl: 'src/UiSwitch.vue'
+    },
+    {
+        path: '/ui-tabs',
+        component: UiTabs,
+        title: 'UiTabs',
+        sourceUrl: 'src/UiTabs.vue'
+    },
+    {
+        path: '/ui-textbox',
+        component: UiTextbox,
+        title: 'UiTextbox',
+        sourceUrl: 'src/UiTextbox.vue'
+    },
+    {
+        path: '/ui-toolbar',
+        component: UiToolbar,
+        title: 'UiToolbar',
+        sourceUrl: 'src/UiToolbar.vue'
+    },
+    {
+        path: '/ui-tooltip',
+        component: UiTooltip,
+        title: 'UiTooltip',
+        sourceUrl: 'src/UiTooltip.vue'
+    }
+];
+
+// Generate a Vue Router compatible routes map from the menu
+const routes = menu.map(menuItem => {
+    return {
+        path: menuItem.path,
+        component: menuItem.component,
+        meta: {
+            title: menuItem.title,
+            sourceUrl: menuItem.sourceUrl
+        }
+    };
+});
+
+const router = createRouter({
```

Comment:
Needed for the active link indicator in the sidebar.

```suggestion
const router = createRouter({
    linkActiveClass: 'is-active',
```